docs/content/docs/design_concepts/osm_components_interactions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "OSM Components & Interations"
+description: "OSM Components & Interations"
+type: docs
+weight: 1
+---
+
+## OSM Components & Interactions
+
+![OSM Components & Interactions](./docs/content/images/osm-components-and-interactions.png)
```

Comment:
I think the correct path may be `/docs/content/docs/images/osm-components-and-interactions.png` but I'm fine with merging and seeing how the build goes.

Reply:
Ah, I see it at https://docs.openservicemesh.io/docs/images/osm-components-and-interactions.png.